packages/@react-aria/interactions/test/usePress.test.js

```diff
@@ -1557,4 +1629,15 @@ describe('usePress', function () {
       ]);
     });
   });
+
+  it('should focus the target even if preventFocusOnPress is true', function () {
+    let {getByText} = render(
+      <Example preventFocusOnPress />
+    );
+
+    let el = getByText('test');
+    fireEvent.click(el);
+
+    expect(document.activeElement).toBe(el);
+  });
```

Comment:
I didn't apply `preventFocusOnPress` to the virtual/screenreader click case, felt like that case shouldn't be affected by that prop. Perhaps `focusWithoutScrolling` should be removed from the virtual click case since iOS 14 VoiceOver focuses the element targeted by the voiceover cursor (I think you mentioned this yesterday @devongovett, correct me if I misremembered)

Reply:
Oh missed this. I don't think it should be removed yet given iOS 14 usage is probably still low. I think it makes sense for onClick as well? This will be used on things that shouldn't receive focus so I think it makes sense to be consistent in all cases. Can't think of a problem that would cause.